site/docs/master/release.md

```diff
@@ -1,61 +1,77 @@
 # Release
 
-## Preparing a new release
+## Preparing release version
```

Comment:
Did you want to include these changes in the v0.19.0 docs as well?

Reply:
Good catch, I need to regenerate the docs! Thanks.